wlib/PreferenceDlg.cpp

```diff
@@ -552,6 +553,9 @@ class CPrefEditorDlg : public CDialogImpl<CPrefEditorDlg>,
 		m_config = config;
 		m_TabWidth = m_config->Get(GLOBAL_TAB_WIDTH);
 		m_TabUseSpaces = m_config->Get(GLOBAL_TAB_USESPACES);
+		m_BOM = m_config->Get(GLOBAL_BOM);
```

Comment:
You can supply a default value in the Get call (it should default to on), so no need for the `if` statement